.github/workflows/prototyping.yml

```diff
@@ -0,0 +1,55 @@
+name: prototyping
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'prototyping'
```

Comment:
We should probably remove this now since this branch is from a different repository and presumably was used for testing

Reply:
Done.